src/components/Filter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+interface Props {
+  todos: Todo[],
+  filter: Filter,
+  setFilter: (filter: Filter) => void;
+}
+
+export const FilterTodo: React.FC<Props> = ({ todos, filter, setFilter }) => (
+  <footer className="todoapp__footer" data-cy="Footer">
```

Comment:
In checklist is written that Code should be split into small, reusable components, like Filter and so on. So we can reuse it. But you have big Filter component attached to the footer. What if we will want to put Filter logic into another section?